tmd/view/plot.py

```diff
@@ -93,6 +93,7 @@ def persistence_image(ph, new_fig=True, subplot=111, xlims=None, ylims=None,
     if xlims is None or xlims is None:
         xlims, ylims = analysis.get_limits(ph, coll=False)
 
+    # pylint: disable=unexpected-keyword-arg
```

Comment:
There is something fishy here. But I'll let you fix it in a separate PR

Reply:
Yes, the plotting parts are not very tested, stable, that's why they are optional.